enhancements/storage/csi-driver-install.md

```diff
@@ -116,21 +147,28 @@ in openshift-aws-ebs-csi-driver namespace / openshift-manila-csi-driver.
 1. During upgrade, CVO starts new 4.6 cluster-storage-operator (CSO).
 2. 4.6 CSO detects that there is AWS / Manila driver installed by OLM.
 3. CSO deletes Subscription of the operator. OLM removes the operator, but leaves the CRD, CR and the driver / operand running.
-4. CSO runs the new CSI driver operator.
-5. The CSI driver operator adopts the old operand - it reconciles the old CSI driver objects (Deployment, DaemonSet,
-   RBAC, ...) with updated 4.6 content. I.e., all objects created by the 4.6 operator must have the same name
-   as the objects created by the old 4.5 operator.
+4. CSO deletes the old namespace (`openshift-aws-ebs-csi-driver` / `openshift-manila-csi-driver`).
```

Comment:
What about other cluster-scoped objects, like RBAC rules and cloud credentials (they have a reference to the operand namespace)? Should we delete them or just expect the new operator to resync and change the namespaces they reference to?

IMO it's better to follow the old namespace approach, i.e., delete them.

 created in `openshift-cloud-credential-operator` namespace?

Reply:
Added note about cluster-scoped object. Delete everything except for StorageClass and CR.